### ISSUE TITLE
Ordering fails when using polymorphic collection of objects

### ISSUE THREAD

**Issue by BtStSoft** [django-ordered-model/django-ordered-model#71]
When an ordered list contains various different models, all subclassed from a base model (which is a subclass of an OrderedModel) the order is managed per subclass, not for the complete list (base model class)
Reason is the self.class reference in the method get_ordering_queryset (line 34 in models.py)

Would be great if you could set a classname to use for the ordering somehow. In the example below one would set that classname to 'Question'.

Simplified example: an ordered list of various different kind of questions
```

class Question(OrderedModel):
    question = models.TextField()
    class Meta:
        ordering = ('order',)

class MultipleChoiceQuestion(Question):
    good_answer = models.TextField()
    wrong_answer1 = models.TextField()

class OpenQuestion(Question):
    answer = models.TextField()

```
Problem: Create two questions, one of each kind. They both obtain order=0.

**Comment by mcwebb** [comment]
@BtStSoft did you find a way round this? Or a different ordering django module that works with subclasses?
I have exactly the same issue.

**Comment by emontellese** [comment]
I found a way around this for my "Event" abstract base model by creating an "OrderedEvent" (which is derived from OrderedModel.  OrderedModel then uses a GenericForeignKey() to point to the derived Sub-Classes.

caveat... I'm new to Django, so there could be some very good reason not to do this that I don't see yet (and if so, please let me know).

**Comment by sulazix** [comment]
Hello, I have the same isssue with polymorphic relation, can you post your code to get it working ?

**Comment by BtStSoft** [comment]
My changes can be found on: https://github.com/bfirsh/django-ordered-model/pull/72/commits/e6f4407e43acad176d46b7559deefc1004801e1b

**Comment by 30f0fn** [comment]
that's wild... I came across this thread while trying to subclass MultipleChoiceQuestion(Question) after Question(OrderedModel).

**Comment by shuckc** [comment]
This was fixed way back in #116, closing issue
